CustomerTicketClose/Kernel/Modules/CustomerTicketClose.pm

```diff
@@ -1,115 +1,110 @@
-# --
-# Kernel/Modules/CustomerTicketClose.pm - close a ticket
-# Copyright (C) 2012-2016 tuxwerk - http://www.tuxwerk.de
-# --
-# This software comes with ABSOLUTELY NO WARRANTY. For details, see
-# the enclosed file COPYING for license information (AGPL). If you
-# did not receive this file, see http://www.gnu.org/licenses/agpl.txt.
-# --
-# --
-
-package Kernel::Modules::CustomerTicketClose;
-
-use strict;
-use warnings;
-
-use Kernel::System::State;
-use Kernel::System::Ticket;
-use Kernel::System::Time;
-
-sub new {
+# -- Kernel/Modules/CustomerTicketClose.pm - close a ticket Copyright (C) 2012-2016 tuxwerk - http://www.tuxwerk.de -- This software comes with ABSOLUTELY NO WARRANTY. For details, see the enclosed file COPYING for license 
```

Comment:
The whitespace changes in this file seem somewhat odd... Could you fix it? Otherwise it looks fine.